installer/manifests/keptn/values.yaml

```diff
@@ -1,8 +1,294 @@
-control-plane:
+continuousDelivery:
```

Comment:
except for the 2 other comments from this file, everything else in this file was just merged together from control plane and cont.deliv.